Spanner/src/Operation.php

```diff
@@ -224,7 +230,13 @@ public function execute(Session $session, $sql, array $options = [])
 
         $context = $this->pluck('transactionContext', $options);
 
-        $call = function ($resumeToken = null) use ($session, $sql, $options) {
+        // Initially with begin, transactionId will be null.
+        // Once generated it will get populated from Result->rows()
+        // Incase of stream failure, the transactionId is preserved here to reuse.
+        $call = function ($resumeToken = null) use ($session, $sql, $options, &$transactionId) {
+            if ( !is_null($transactionId) and !is_array($transactionId) ) {
+                $options['transaction'] = ['id' => $transactionId];
```

Comment:
If this is all the argument is doing, we can document to our users that this is how a transaction ID can be utilized. It doesn't seem like we need to make any changes to the library... e.g.

```php
// this is already supported
$operation->execute($session, ['transaction' => ['id' => $transactionId]]);

// this is the functionality we are adding, which is not significantly different
$operation->execute($session, [], $transactionId);

// if we really want to make this easier / more readable, we could add support for 
// `transactionId` in the $options array
$operation->execute($session, ['transactionId' => $transactionId]);
```